### DISCUSSION TITLE
Is there some proxy mechanism to achieve the migration of using the native canvas?

### DISCUSSION THREAD

**Discussion by BingCoke** [fabricjs/fabric.js#9995] [Q&A]
This is a very wonderful library, but I have already written a lot of code with the native canvas based on the context. Are there any proxy mechanisms that can facilitate the migration to the use of the native canvas? Because I hope to be able to listen for events on the components I draw.

**Comment by asturur** [comment]
This question is so generic that the answer can be just no. It depends on the code you wrote and what kind of app you are doing and what functionalities you are looking for.
